src/pages/our-mission.jsx

```diff
@@ -8,127 +9,98 @@ import "../css/Index.css"
-                            Our platform is open source, and can be modified and used by any non-profit organisation
-                            around the world to improve access to justice in their country.
-                        </p>
-                    </div>
-
-                    <div id="our-data" className="content-section">
-                        <h2 className="body-title">Our data</h2>
-                        <p>Because case law is not subject to copyright restrictions, we have been able to re-purpose
-                            PDFs of case law from the Ministry of Justice websites and convert them into usable pieces
-                            of data that a computer can read and search. </p>
-                        <p>
-                            The case law data we have currently comprises High Court, Court of Appeal and Supreme Court
-                            decisions from present day back to approximately 2004 (with some exceptions, for example
-                            some suppressed decisions and other judgments that have not been published), plus ACC appeal
-                            decisions in the District Court.
-                        </p>
-                        <p>We look at those PDF files and analyse them to see what pieces of information might be useful
-                            – for example, references to pieces of legislation, dates, relationships and citations.
-                            We’re always working on expanding the data we can extract. </p>
-
-                    </div>
-                    <div id="stewardship" className="content-section">
-                        <h2 className="body-title">Stewardship</h2>
-                        <p>We want to make legal information as accessible as we can.</p>
-
-                        <p>That means our data is open and accessible to everybody. You can interact with our data
-                            through our website and search. Or you can tap directly into our database through our API.
-                            All of our code is open source and licensed to the public under the GPL open source license.
-                        </p>
+            />
+            <HeroSmall title="Our Mission" />
+            <div className="inner">
+                <div className="body-wrap content-page right-on-top">
+                    <div className="body-left">
+                        {
+                            missionItems.map(({ title, data, text }, idx) => (
+                                <article id={title} className="content-section" key={idx}>
+                                    <h2 className="body-title">{title}</h2>
+                                    <p className="blog-meta">{data}</p>
+                                    {
+                                        text.map(({ content_html }, idx) => (
+                                            <p key={idx} dangerouslySetInnerHTML={{ __html: content_html }} />
```

Comment:
We'll want to get some sanitisation on this before we take it to production. Will get onto this tomorrow as I've got some free time.

Reply:
I'd like the sanitisation before we go to prod please.